tpr/tpr.go

```diff
@@ -26,17 +26,15 @@ type Config struct {
 
 	// Settings.
 
-	// Name of the kind of ThirdPartyObjects. It should be in lower case
-	// and hyphen delimited. Kind names will be converted to CamelCase
-	// when creating ThirdPartyObjects. Hyphens in the kind are assumed to
-	// be word breaks. For instance the kind camel-case would be converted
-	// to CamelCase but camelcase would be converted to Camelcase.
+	// Name takes the form <kind>.<domain>. The domain is also known as
```

Comment:
Can we settle on one of domain and group? What's the benefit of mentioning both?